lib/declaration.ts

```diff
@@ -439,6 +446,22 @@ class DeclarationTranspiler extends base.TranspilerBase {
       }
     }
   }
+
+  private visitCallSignature(name: string, signature: ts.SignatureDeclaration,
+                             typeParameters: ts.NodeArray<ts.TypeParameterDeclaration>) {
+    this.emit('typedef');
+    if (signature.type) {
```

Comment:
What would this be here? If I understand this correctly, it'd be the resolved type represented by the interface declaration; I don't think we'd want to visit that here, would we?